docs/external_contribution_request.md

```diff
@@ -0,0 +1,17 @@
+## Issue title
+
+Publish `EXTENSION_NAME` to Open VSX
+
+## Issue body
+
+```md
+Unfortunately MS prohibits usages of MS marketplace by any other products or redistribution vsix files from it. Because of it we kindly ask you to take the ownership of your namespace in [OpenVSX](https://open-vsx.org/) and publish there as well.
+
+OpenVSX is a vendor neutral alternative to MS marketplace used by all other derivatives of VS Code like [VSCodium](https://vscodium.com/), [Gitpod](gitpod.io), [OpenVSCode](https://github.com/gitpod-io/openvscode-server), [Theia](https://theia-ide.org/) based IDEs, and so on.
+
+The docs for publishing the extension are [here](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions). Technically this should be straightforward. You get a token and call `ovsx publish` with it. For example, this is how RedHat publishes their Java extension: [redhat-developer/vscode-java@ff51dbf/Jenkinsfile#L73-L82](https://github.com/redhat-developer/vscode-java/blob/5d4f8d58b8e919534800ca7bc41e8513c288f573/Jenkinsfile#L78-L82)
+
+[Here](https://github.com/PeterWone/vsc-print/pull/121) is also an example PR that contributes a release GitHub Action which can publish to GitHub, MS and OpenVSX at the same time. Let @filiptronicek or @akosyakov know if you are open to a PR contributing such GitHub Action. They would gladly help out, even if there are just some questions or suggestions.
+
+Gitpod as well supports open-source developers by providing unlimited access to their platform. Feel free to reach out via https://www.gitpod.io/docs/professional-open-source

```

Comment:
I'm not so sure on the Gitpod plug in this template, as we don't want Gitpod to be mistaken as the direct owner of Open VSX. I also wouldn't necessarily mention Anton/yourself. 

```suggestion
Dear extension author,

Please publish this extension to the Open VSX marketplace. 

**Context**
Unfortunately, as Microsoft prohibits usages of Microsoft marketplace by any other products or redistribution vsix files from it, in order to use VS Code extensions in non-Microsoft products, we kindly ask that you take ownership of the VS Code extension namespace in [OpenVSX](https://open-vsx.org/) and publish this extension on Open VSX.

**What is OpenVSX? Why does it exist?**
OpenVSX is a vendor neutral alternative to MS marketplace used by all other derivatives of VS Code like [VSCodium](https://vscodium.com/), [Gitpod](gitpod.io), [OpenVSCode](https://github.com/gitpod-io/openvscode-server), [Theia](https://theia-ide.org/) based IDEs, and so on.

**How can you publish to Open VSX?**
The docs to publish an extension can be found [here](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions). This process is straightforward and shouldn't take too long. Essentially, you need an authentication token and to execute the command `ovsx publish` to publish your plugin. See this example from RedHat publishes their Java extension: [redhat-developer/vscode-java@ff51dbf/Jenkinsfile#L73-L82](https://github.com/redhat-developer/vscode-java/blob/5d4f8d58b8e919534800ca7bc41e8513c288f573/Jenkinsfile#L78-L82)

You can also find [an example PR](https://github.com/PeterWone/vsc-print/pull/121) that contributes a release GitHub Action which publishes to GitHub, MS and OpenVSX at the same time. 
```

Reply:
This looks like a great change, @loujaybee! I would go so far as to add a section linking to the https://github.com/open-vsx/publish-extensions/blob/ft/issue-and-pr-templates/docs/direct_publish_setup.md doc and add a mention of us (or the person that creates the PR) being open to contributing the CI workflow to the target repository.

The idea with the mentions was by @akosyakov, so he probably has some thoughts 😃.

Reply:
I'm fine to drop it. Put Gitpod there to motivate author a bit to become OpenVSX users through it.

Reply:
> I'm not so sure on the Gitpod plug in this template, as we don't want Gitpod to be mistaken as the direct owner of Open VSX. I also wouldn't necessarily mention Anton/yourself.

I agree. As a possible alternative, I am happy to collaborate on a resource that points people to ways they can try Open VSX. One idea is a page on open-vsx.org, a la https://github.com/EclipseFdn/open-vsx.org/issues/773